georocket-server/src/main/java/io/georocket/ImporterVerticle.java

```diff
@@ -191,7 +227,8 @@ protected void onImport(Message<JsonObject> msg) {
           });
         })
         .last() // "wait" for last event (i.e. end of file)
```

Comment:
You could use `count()` instead of `wait()` here. This would eliminate `countChunks`.